### PR TITLE
Added conda recipe for babelizer v0.2.0

### PR DESCRIPTION
**Pull request by mcflugen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/babelizer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    - python >=3.8
+    - pip
+    - black
+    - click
+    - gitpython
+    - pyyaml
+    - tomlkit
+    - isort >=5
+    - jinja2
+    - cookiecutter
```
Comment (scopatz) [inline recipes/babelizer/meta.yaml:31]: Are all of these needed to run pip?

Comment (mcflugen) [reply recipes/babelizer/meta.yaml:31]: These are all dependencies of the package, but I suppose not needed to just run pip. Does this mean they should only be in the *run* section?

Comment (mcflugen) [reply recipes/babelizer/meta.yaml]: @scopatz I've removed these requirements from the *host* section, but left them in the *run* section. I hope that's what you were getting at.

Thank you for looking at this so quickly! It's very much appreciated.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**